.travis.yml

```diff
@@ -21,6 +21,7 @@ before_install:
   - if [[ "$BOARD" =~ "arduino:samd:" ]]; then
       arduino --install-boards arduino:samd;
       arduino --install-library WiFi101;
```

Comment:
@tameraw The `Serial.setDebugOutput(true)` is only for 8266 board. It is an interface defined in the 8266 board package, not for all boards.

You can ref these pages:
http://www.panstamp.org/forum/archive/index.php/thread-4541.html 
http://arduino.esp8266.com/versions/1.6.5-1044-g170995a/doc/reference.html#serial